src/Post/Post.js

```diff
@@ -411,11 +414,13 @@ function MediaSection({ post, isExpanded, dimensions }) {
                 active={mediaIndex === index}
                 fraction={1 / items.length}
                 aspectRatio={aspectRatio}
+                setIsLoading={setIsLoading}
+                isLoading={isLoading}
```

Comment:
This will cause issues as all elements will fire this event, if the 3rd image loaded before the first, you'll stop the animation. We need to only add these for the first item.